srcpkgs/python3-Flask/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'python3-Flask'
 pkgname=python3-Flask
-version=1.1.2
-revision=3
+version=2.0.1
+revision=1
 wrksrc="${pkgname#*-}-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-Jinja2 python3-Werkzeug
  python3-itsdangerous python3-click"
-checkdepends="${depends}
+checkdepends="${depends} python3-pip python3-wheel
```

Comment:
`pip` in checkdepends means it's installing external packages and not testing with the void ones. What is it trying to install with pip?